integration-tests/deployment/crib/envsetup/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc/credentials"
+
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment"
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment/devenv"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+)
+
+var (
+	rootCmd       = &cobra.Command{Use: ""}
+	lggr          logger.Logger
+	cribConfig    string
+	cribEnv       *deployment.Environment
+	cribEnvConfig devenv.EnvironmentConfig
+)
+
+func init() {
+	var closeLggr func() error
+	lggr, closeLggr = logger.NewLogger()
+	defer func() {
+		err := closeLggr()
+		if err != nil {
+			panic(err)
+		}
+	}()
+	rootCmd.PersistentFlags().StringVarP(&cribConfig, "crib-config", "c", "", "CRIB environment configuration file")
+	err := rootCmd.MarkPersistentFlagRequired("crib-config")
+	if err != nil {
+		lggr.Fatalw("no chain configuration file provided", "err", err)
+	}
+	cribEnvConfig, err = devenv.LoadEnvironmentConfig(cribConfig)
+	if err != nil {
+		lggr.Fatalw("failed to load environment configuration", "err", err)
+	}
```

Comment:
that logic should probably go under command implementation and not init function. 
init function should be just for initializing cobra flags etc. It won't have access to the cribConfig variable here.